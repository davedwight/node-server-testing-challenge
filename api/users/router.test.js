const db = require("../../data/db-config");

test("is the correct environment", () => {
  expect(process.env.DB_ENV).toBe("testing");
});
