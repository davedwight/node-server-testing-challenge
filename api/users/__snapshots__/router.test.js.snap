// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[DELETE] snapshot test 1`] = `
Object {
  "user_id": 3,
  "username": "baz",
}
`;

exports[`[GET] /api/users snapshot test 1`] = `
Array [
  Object {
    "user_id": 1,
    "username": "foo",
  },
  Object {
    "user_id": 2,
    "username": "bar",
  },
  Object {
    "user_id": 3,
    "username": "baz",
  },
]
`;

exports[`[POST] /api/users snapshot test 1`] = `
Object {
  "user_id": 4,
  "username": "foobar",
}
`;
